pkg/commands/app.go

```diff
@@ -28,7 +28,14 @@ func NewApp(version string, commitSHA string) *cobra.Command {
 }
 
 func NewRootCommand() *cobra.Command {
-	usageTemplate := `Curio - discover sensitive data flows and security risks.
+	usageTemplate := `
+  [0;1;35;95m▄▄▄[0m                  [0;37m▀[0m
+[0;35m▄▀[0m   [0;35m▀[0m [0;35m▄[0m   [0;35m▄[0m   [0;37m▄[0m [0;37m▄▄[0m  [0;37m▄▄▄[0m     [0;37m▄▄[0;1;30;90m▄[0m
+[0;35m█[0m      [0;37m█[0m   [0;37m█[0m   [0;37m█▀[0m  [0;37m▀[0m   [0;1;30;90m█[0m    [0;1;30;90m█▀[0m [0;1;30;90m▀█[0m
+[0;37m█[0m      [0;37m█[0m   [0;37m█[0m   [0;1;30;90m█[0m       [0;1;30;90m█[0m    [0;1;30;90m█[0m   [0;1;35;95m█[0m
+ [0;37m▀▄▄▄▀[0m [0;1;30;90m▀▄▄▀█[0m   [0;1;30;90m█[0m     [0;1;30;90m▄▄[0;1;35;95m█▄▄[0m  [0;1;35;95m▀█▄█▀[0m
+
+discover sensitive data flows and security risks.
```

Comment:
```suggestion
Discover sensitive data flows and security risks.
```
Given it's no longer part of the tagline with the title, I think it looks less out of place capitalized.